config/locales/views/groups/en.yml

```diff
@@ -27,6 +27,7 @@ en:
     empty: Empty Group
     grace_day_over_limit: You have assigned one or more students who has less grace credits than the amount already used by group %{group} for this assignment.
     group_name_already_in_use: This name is already in use for this assignment.
+    group_name_already_in_use_diff_assignment: This name is already in use for a different assignment.
```

Comment:
Following my above comment, this error message doesn't quite communicate the right idea. The problem it should convey is something like "You're trying to rename to an existing group, which isn't allowed because you've already submitted a file." Please work off of that to create a new version of this error message.